src/lib/types.d.ts

```diff
@@ -1,38 +1,54 @@
+import { type ITenantContract } from "@retro-pgf/contracts";
+
+export type TenantNamespace = "optimism";
+
+export type TenantContracts = {
+  alligator?: ITenantContract;
+  governor: ITenantContract;
+  proposalTypesConfigurator: ITenantContract;
```

Comment:
`proposalTypesConfigurator` is optional. At this point it's OP specific contract. All DAOs will only have `governor` and `token` as mandatory